tests/test_hyper_block.py

```diff
@@ -2,8 +2,9 @@
 
 from autokeras.auto.auto_model import *
 from autokeras import const
-from autokeras.hypermodel.hyper_block import XceptionBlock
-from autokeras.hypermodel.hyper_head import ClassificationHead
+
+from autokeras.hypermodel.hyper_block import XceptionBlock, RNNBlock
```

Comment:
To follow the keras style guide, the imports should be only importing modules and one module per line.

`from autokeras.hypermodel import hyper_block
from autokeras.hypermodel import hyper_head
...
`

Reply:
Similar import statements are present in hyper_block.py as well. Fixing them too.

Just to clarify, do you really prefer imports to be like :
`from autokeras.hypermodel import hyper_block`

or is it fine if I do like below
`from autokeras.hypermodel.hyper_block import *`

In first case, I must replace all occurrences with _hyper_block.XceptionBlock_  or _hyper_block.RNNBlock_. 
The second case doesn't require that (changes will happen only to import statements)